docs/CIPHERS.md

```diff
@@ -461,6 +461,14 @@ specific ciphersuites. They are
 [defined](https://docs.microsoft.com/windows/desktop/SecCrypto/alg-id) by
 Microsoft.
 
+There is also the case that the selected algorithm will not match the ciphers
+offered by the server during the SSL negotiation. In this case curl will 
+print an `CURLE_SSL_CONNECT_ERROR (35 SEC_E_ALGORITHM_MISMATCH` and the
```

Comment:
I think you meant (35) SEC_E_ALGORITHM_MISMATCH?

Reply:
Yes.. Doing a force push to update it